contracts/destination/xreceivers/AuthForwarderXReceiver.sol

```diff
@@ -0,0 +1,229 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import {IConnext} from "@connext/interfaces/core/IConnext.sol";
+import {IXReceiver} from "@connext/interfaces/core/IXReceiver.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+abstract contract AuthForwarderXReceiver is IXReceiver, Ownable {
```

Comment:
note: our contracts generally use `ProposedOwnable`, but since this is an abstract contract we should likely use the regular interface.

should document how to change ownership though